.github/workflows/tag.yml

```diff
@@ -31,11 +32,54 @@ jobs:
         id: create_tag
         uses: mikefarah/yq@v4
         with:
-          cmd: yq -r '"v" + (.MAJ | tostring) + "." + (.MIN | tostring)' ${{ steps.workflow_directory.outputs.name }}/VERSION.yaml
+          cmd: yq -r '"v" + (.MAJ | tostring) + "." + (.MIN | tostring) + "." + (.PATCH | tostring)' ${{ steps.workflow_directory.outputs.name }}/VERSION.yaml
```

Comment:
addition of `(.PATCH ...` captures the forthcoming patch versions from https://github.com/wildlife-dynamics/ecoscope-workflows/pull/1022